src/harness/reference_models/ppa_gwpz_fss_gwbl_purge/ppa_gwpz_fss_gwbl_purge_test.py

```diff
@@ -0,0 +1,94 @@
+#    Copyright 2017 SAS Project Authors. All Rights Reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+
+import json
+import os
+import unittest
+import full_activity_dump
+from util import makePpaAndPalRecordsConsistent
+from reference_models.ppa_gwpz_fss_gwbl_purge import ppa_gwpz_fss_gwbl_purge
+from reference_models.pre_iap_filtering import pre_iap_util
+class TestFssPurge(unittest.TestCase):
+
+  def test_ppaGwpzFssPlusGwblPurgeReferenceModel_default(self):
+    
+   # TEST DATA
+   cbsd_0 = json.load(
+     open(os.path.join('testdata', 'cbsd_0.json')))
+   cbsd_1 = json.load(
+     open(os.path.join('testdata', 'cbsd_1.json')))
+   cbsd_2 = json.load(
+     open(os.path.join('testdata', 'cbsd_2.json')))
+   cbsd_3 = json.load(
+     open(os.path.join('testdata', 'cbsd_3.json')))
+   cbsd_4 = json.load(
+     open(os.path.join('testdata', 'cbsd_4.json')))
+   cbsd_5 = json.load(
+     open(os.path.join('testdata', 'cbsd_5.json')))
+   # Load PPA record
+   gwpz_record = json.load(
+     open(os.path.join('testdata', 'gwpz_0.json')))
+   ppa_record = json.load(
+     open(os.path.join('testdata', 'ppa_0.json')))
+   pal_record = json.load(
+     open(os.path.join('testdata', 'pal_0.json')))
+   fss_record = json.load(
+     open(os.path.join('testdata', 'fss_0.json')))
+   gwbl_record = json.load(
+     open(os.path.join('testdata', 'gwbl_0.json')))
+
+   pal_low_frequency = pal_record['channelAssignment']['primaryAssignment']['lowFrequency']
+   pal_high_frequency = pal_record['channelAssignment']['primaryAssignment']['highFrequency']
+   ppa_record, pal_records = makePpaAndPalRecordsConsistent(ppa_record,
+                                                                [pal_record],
+                                                                pal_low_frequency,
+                                                                pal_high_frequency,
+                                                                'test_user_1')
+
```

Comment:
Don't forget to add something to the cluster list of the PPA.

Reply:
Modified the test file to include testing for PPA cluster list.